vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -192,8 +192,12 @@ public String getValue() {
      * @return the sanitized {@code htmlValue} property from the webcomponent.
      */
     public String getHtmlValue() {
+        String htmlValueString = getHtmlValueString();
+        if (htmlValueString != null) {
+            htmlValueString = sanitize(htmlValueString);
+        }
         // Using basic whitelist and adding img tag with data protocol enabled.
```

Comment:
This comment is no longer in the right place, and it's a bit unsettling that this method can return a value from getHtmlValueString() that isn't run through sanitize; at a quick glance it looks scary. Could we change the block to be a guard block instead:

```
        String htmlValueString = getHtmlValueString();
        if (htmlValueString == null) {
          return null;
        }
        // Using basic whitelist and adding img tag with data protocol enabled.
        return sanitize(htmlValueString);
```

Reply:
Valid point, added guard block and also updated the doc for the method.